<launch>
  <!-- <node name="controllInterface" pkg="head_pkg" type="controllInterfaceRev2.py" output="screen" launch-prefix="sudo"/> -->
  <node name="neckInterface" pkg="head_pkg" type="NeckInterface.py" output="screen"/>
</launch>
